### PR TITLE
AOM-26: UI improvements and fix location drop down bug

### PR DESCRIPTION
**Pull request by jomadoye**
# JIRA TICKET NAME:
AOM-22: [UI improvements and fix location drop down bug](https://issues.openmrs.org/browse/AOM-26)

## SUMMARY:
This PR ensures that the current location is displayed to the user on the dropdown, as well as other slight UI fixes

### CODE REVIEW

File: app/js/components/common/header.js [hunk 1]
```diff
@@ -105,34 +123,55 @@ export default class Header extends Component {
...
                 ? this.state.currentLocationTag
                 : this.state.defaultLocation}
-              <span className="caret"/>
+              <span className="caret navbar-glyphicon-font"/>
             </a>
-            <ul className="dropdown-menu dropdown-menu-large row">
+            <ul 
+              id="location-ul"
+              className="dropdown-menu dropdown-menu-large row">
               {/*Execute the function*/}
```
Comment (kingisaac95) [inline app/js/components/common/header.js]: I don't think this comment is necessary here

File: app/js/components/common/header.js [hunk 2]
```diff
@@ -105,34 +123,55 @@ export default class Header extends Component {
...
             </ul>
           </li>
+
           <li>
-            <a href={this.state.currentLogOutUrl}>Logout {' '}
-              <span className="glyphicon glyphicon-log-out"/></a>
+            <a 
+              className="navbar-font"
+              href={this.state.currentLogOutUrl}>Logout {' '}
+              <span className="glyphicon glyphicon-log-out navbar-glyphicon-font"/></a>
```
Comment (kingisaac95) [inline app/js/components/common/header.js]: Could you bring the closing `</a>` tag to a new line to match the opening tag?

File: app/css/openmrs-addonmanager.css [hunk 3]
```diff
@@ -323,3 +328,80 @@ span.title.breadcrumb-item {
...
+    background-color: #007fff;
+    color: #fff !important;
+}
+
+.location {
+    border-width: 1px;
+    border-bottom-width:1px;
+    border-bottom-color:rgba(148, 147, 147, 0.41);
+    border-bottom-style: solid;
+    margin-top: 5px !important;
```
Comment (kingisaac95) [inline app/css/openmrs-addonmanager.css]: Can we not have `!important` @andela-jomadoye?

File: app/css/openmrs-addonmanager.css [hunk 4]
```diff
@@ -323,3 +328,80 @@ span.title.breadcrumb-item {
...
+    background-color:#5B57A6;
+    z-index: 1000; 
+ }
+
+.dropdown-menu {
+     padding: 0;
+ }
+
+ .my-account:hover {
+     background-color: #5B57A6 !important;
```
Comment (kingisaac95) [inline app/css/openmrs-addonmanager.css]: Ditto

**Review by kingisaac95 — CHANGES_REQUESTED** [review]

**Comment by kingisaac95** [comment]
Nice job @andela-jomadoye!! 👍

**Comment by wanjikum** [comment]
Looks good 👍

**Comment by paulupendo** [comment]
Good Job @andela-jomadoye

**Pull request merged**